### PR TITLE
Bump to 0.3.12

### PR DESCRIPTION
**Pull request by hahnbeelee**
Includes #71

### CODE REVIEW

**Review by benjamin-shen — APPROVED** [review]

**Comment by hahnbeelee** [comment]
@NicoZweifel Hm odd. Can you see the failed Storybook UI tests?

**Comment by NicoZweifel** [comment]
Yea I was afraid this might happen at some point. I can prepare a PR. It is because of navigator.clipboard which is missing in tests. Ideally we mock the clipboard.

**Comment by NicoZweifel** [comment]
I created a [Hotfix PR](https://github.com/mintlify/components/pull/74) that targets this release branch.
The tests pass in the runner locally now and they should pass in the action.
I added a mock for `navigator.clipboard` with the help of `spyOn`.
It works and was kind of tricky to figure out since old solutions all dont work but I am not 100% sure if this is the right way since jest in storybook seems to be a bit different.

Is there a place to put testing utils? I have some duplicated code in the tests and we could maybe organize the mocks better (if that is possible with storybook-jest).

**Comment by hahnbeelee** [comment]
@NicoZweifel How generic are the testing utils? We can create a package if it's generic enough for other people to benefit from it. 👀 If you want complete ownership over it you can even publish your own repo/npm package for it 👀

**Comment by NicoZweifel** [comment]
@hahnbeelee I think these conflicts might be because of squashing

File: src/stories/Interactive/Code/CodeBlock.stories.tsx [hunk 1]
```diff
@@ -72,28 +72,47 @@ CodeBlockInteractions.args = {
   filename: fileName,
 };
 CodeBlockInteractions.play = async ({ canvasElement }) => {
+  let clipboardData: unknown = '';
```
Comment (benjamin-shen) [inline src/stories/Interactive/Code/CodeBlock.stories.tsx:75]: Can clipboardData be typed as string?

Comment (NicoZweifel) [reply src/stories/Interactive/Code/CodeBlock.stories.tsx:75]: We could but you'd have to define it in the MockImplementation as well and the type is not really required for anything.

**Pull request merged**